[3.4.5] R.style.ClockTimePickerDialog

# Bug report

Hi, I saw that in the last commit you changed how you access ClockTimePickerDialog, it now gives me a build error:

```error: cannot find symbol
                  ? R.style.ClockTimePickerDialog
                           ^
  symbol:   variable ClockTimePickerDialog
  location: class style
```

I mean 5f0c8ee98b13775e7fbb38b902b84d32774b8790

react-native: 0.64.0

--- a/android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java
+++ b/android/src/main/java/com/reactcommunity/rndatetimepicker/RNTimePickerDialogFragment.java
@@ -75,12 +75,16 @@ public class RNTimePickerDialogFragment extends DialogFragment {
       switch (display) {
         case CLOCK:
         case SPINNER:
-          int theme = display == RNTimePickerDisplay.CLOCK
-                  ? R.style.ClockTimePickerDialog
-                  : R.style.SpinnerTimePickerDialog;
+          String resourceName = display == RNTimePickerDisplay.CLOCK
+                  ? "ClockTimePickerDialog"
+                  : "SpinnerTimePickerDialog";
           return new RNDismissableTimePickerDialog(
                   activityContext,
-                  theme,
+                  activityContext.getResources().getIdentifier(
+                          resourceName,
+                          "style",
+                          activityContext.getPackageName()
+                  ),
                   onTimeSetListener,
                   hour,
                   minute,

